src/components/Review.jsx

```diff
@@ -1,20 +1,41 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { fetchReview } from "../api";
+import { fetchReview, updateReviewVotes } from "../api";
 import Comments from "./Comments";
 
 const Review = () => {
   const review_id = useParams().review_id;
   const [review, setReview] = useState({});
   const [isLoading, setIsLoading] = useState(true);
-
+  const [votes, setVotes] = useState("");
+  const [err, setErr] = useState(null);
+  const [hasVoted, setHasVoted] = useState(true);
+  const [inc, setInc] = useState(1);
   useEffect(() => {
     setIsLoading(true);
     fetchReview(review_id).then((data) => {
       setReview(data);
+      setVotes(data.votes);
       setIsLoading(false);
     });
   }, [review_id]);
+
+  const handleClick = () => {
+    if (hasVoted) {
+      setInc(1);
+      setHasVoted(false);
+    } else {
+      setInc(-1);
+      setHasVoted(true);
+    }
```

Comment:
i would say the `setInc` state isn't needed here as you could just use `setVotes((currentVote) => currentVote + inc);` in it's place just changing the `+` where necessary